zio-http/shared/src/main/scala/zio/http/codec/internal/Atomized.scala

```diff
@@ -25,30 +25,34 @@ private[http] final case class Atomized[A](
   path: A,
   query: A,
   header: A,
+  headerCustom: A,
   content: A,
 ) {
   def get(tag: HttpCodec.AtomTag): A = {
     tag match {
-      case HttpCodec.AtomTag.Status  => status
-      case HttpCodec.AtomTag.Path    => path
-      case HttpCodec.AtomTag.Content => content
-      case HttpCodec.AtomTag.Query   => query
-      case HttpCodec.AtomTag.Header  => header
-      case HttpCodec.AtomTag.Method  => method
+      case HttpCodec.AtomTag.Status       => status
+      case HttpCodec.AtomTag.Path         => path
+      case HttpCodec.AtomTag.Content      => content
+      case HttpCodec.AtomTag.Query        => query
+      case HttpCodec.AtomTag.Header       => header
+      case HttpCodec.AtomTag.HeaderCustom => headerCustom
+      case HttpCodec.AtomTag.Method       => method
     }
   }
 
   def update(tag: HttpCodec.AtomTag)(f: A => A): Atomized[A] = {
     tag match {
-      case HttpCodec.AtomTag.Status  => copy(status = f(status))
-      case HttpCodec.AtomTag.Path    => copy(path = f(path))
-      case HttpCodec.AtomTag.Content => copy(content = f(content))
-      case HttpCodec.AtomTag.Query   => copy(query = f(query))
-      case HttpCodec.AtomTag.Header  => copy(header = f(header))
-      case HttpCodec.AtomTag.Method  => copy(method = f(method))
+      case HttpCodec.AtomTag.Status       => copy(status = f(status))
+      case HttpCodec.AtomTag.Path         => copy(path = f(path))
+      case HttpCodec.AtomTag.Content      => copy(content = f(content))
+      case HttpCodec.AtomTag.Query        => copy(query = f(query))
+      case HttpCodec.AtomTag.Header       => copy(header = f(header))
+      case HttpCodec.AtomTag.HeaderCustom => copy(headerCustom = f(header))
```

Comment:
It's painful to see Header and HeaderCustom when all the other cases just have a single codec!